src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
You already have the `/` in the api call.
```suggestion
export const BASE_URL = 'https://mate.academy/students-api';
```